scripts/migrations/2020-10-miro-migration/transfer_packager.py

```diff
@@ -0,0 +1,195 @@
+import concurrent
+import itertools
+import os
+import shutil
+import uuid
+
+import attr
+from tqdm import tqdm
+
+from s3 import get_s3_content_length
+
+S3_DOWNLOAD_CONCURRENCY = 3
+
+
+@attr.s
+class TransferPackage:
+    local_location = attr.ib()
+    content_length = attr.ib()
+    s3_location = attr.ib(default=None)
+
+
+def _check_local_content_length(file_location, expected_content_length):
+    """
+    Check that a file has the expected content length in bytes
+    """
+    local_content_length = os.path.getsize(file_location)
+
+    assert local_content_length > 0, "Content length is zero: " f"{file_location}"
+
+    assert local_content_length == expected_content_length, (
+        "Content length mismatch "
+        f"({local_content_length} != {expected_content_length}): "
+        f"{file_location}"
+    )
```

Comment:
Ditto the S3 function above: `_check_file_size_matches()` makes more sense than talking about Content-Length.